app/scripts/controllers/transactions/index.js

```diff
@@ -324,9 +330,9 @@ export default class TransactionController extends EventEmitter {
     this.txStateManager.wipeTransactions(address);
   }
 
+  /* eslint-disable */
   /**
    * Add a new unapproved transaction
-   *
```

Comment:
You need to re-add this line for linting. If I am not mistaken we have the `jsdoc/newline-after-description` rule enabled